### ISSUE TITLE
June 2 discussion around terminology

### ISSUE THREAD

**Issue by kimdhamilton** [w3c-ccg/community#128]
replacing whitelist/blacklist with allow/deny. The topic came up at the tail end of the discussion and we didn't get to address. Discuss on a future call

**Comment by jandrieu** [comment]
I agree with the shift. Where do we use those terms?

**Comment by kimdhamilton** [comment]
It was during the 6/2 call. I noticed the discussion happening in [IRC at the end of the call](https://w3c-ccg.github.io/meetings/2020-06-02/index.html#137) and didn't have time to properly address. It's worth bringing up on a future call.

I looked to see if this topic is mentioned in the new [PWETF guidelines](https://w3c.github.io/PWETF), but didn't see any reference. It would be nice to have a more comprehensive reference, but we can start here. I note that [rails recently addressed this as well](https://github.com/rails/rails/issues/33677). In general, the most common replacement seems to be allowlist/denylist. It has the advantage of being clearer to non-native speakers as well.
